### PR TITLE
Move coordinates schemas to asdf-coordinates-schemas

### PR DESCRIPTION
**Pull request by eslavich**
We've created an independent package for the coordinates schemas, asdf-coordinates-schemas.  This package will reference the schemas from there.

### CODE REVIEW

**Comment by mcara** [comment]
I am so tempted to approve this just because of `+19 −1,054`.

File: setup.cfg [hunk 1]
```diff
@@ -21,6 +21,7 @@ install_requires =
     astropy
     asdf>=2.8.0
     numpy
+    asdf-coordinates-schemas @ git+https://github.com/eslavich/asdf-coordinates-schemas.git@eslavich-create-package
```
Comment (mcara) [inline setup.cfg:24]: I assume this will be updated after https://github.com/asdf-format/asdf-coordinates-schemas/pull/1 is merged and package released.

**Review by mcara — APPROVED** [review]
LGTM, makes sense and tests are passing

**Review by nden — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (eslavich) [reply setup.cfg]: Yes!  And it done!

**Pull request merged**